modules/messages/tests/client/components/Inbox.component.tests.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import '@/config/client/i18n';
+import Inbox from '@/modules/messages/client/components/Inbox.component';
+
+import * as api from '@/modules/messages/client/api/messages.api';
+
+jest.mock('@/modules/messages/client/api/messages.api');
+
+afterEach(() => jest.clearAllMocks());
+
+const me = {
+  _id: '6e1719d8a8098fb2ce44160b',
+  public: true,
+  username: 'testuser',
+  displayName: 'test user',
+};
+
+const threads = [
+  {
+    _id: '5e1719d8a8098fb2ce44160a',
+    read: true,
+    updated: '2020-01-09T12:17:28.967Z',
+    message: {
+      excerpt: 'a nice message',
+    },
+    userFrom: {
+      _id: '5de7ec33f333ce7b23cdfa26',
+      username: 'admin1',
+      displayName: 'Newell Jones',
+    },
+    userTo: {
+      _id: '5de7ec34f333ce7b23cdfcd2',
+      username: 'madyson',
+      displayName: 'Madyson Morissette',
+    },
+  },
+  {
+    _id: '5de7ec8bb633f67cf0dbc265',
+    read: true,
+    updated: '2020-01-02T21:52:04.154Z',
+    message: {
+      excerpt: 'why does that go funny?',
+    },
+    userFrom: {
+      _id: '5de7ec33f333ce7b23cdfa26',
+      username: 'admin1',
+      displayName: 'Newell Jones',
+    },
+    userTo: {
+      _id: '5de7ec34f333ce7b23cdfc61',
+      username: '571eloyhintz',
+      displayName: 'Eloy Hintz',
+    },
+  },
+];
```

Comment:
Would like to create some reusable generators for this kind of test data.

Reply:
There are a few at https://github.com/Trustroots/trustroots/blob/1bb2908072257cd737d66cbe13539d6ab1eab513/testutils/server/data.server.testutil.js